es-MX/firefox/whatsnew/whatsnew-account.ftl

```diff
@@ -2,12 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-
 ### URL: https://www-dev.allizom.org/firefox/60.0/whatsnew/all/
 
-
 ## Required
 
-whatsnew-account-main-headline = No se requiere una cuenta. Pero tal vez quieras una.
-whatsnew-account-main-lead-in = El navegador { -brand-name-firefox } evita coleccionar tantos datos sobre ti que ni siquiera requerimos tu correo, pero cuando lo usas para crear una cuenta { -brand-name-firefox }, nos permite protegerte la privacia en más lugares de tu vida virtual.
-whatsnew-account-main-button = Iniciar sesión
+whatsnew-account-main-headline = No se requiere de una cuenta. Pero tal vez quieras una.
+whatsnew-account-main-lead-in = El navegador { -brand-name-firefox } recopila tan pocos datos sobre ti, que ni siquiera necesitamos tu dirección de correo electrónico. Aun así, cuando lo usas para crear una cuenta { -brand-name-firefox }, podemos proteger tu privacidad durante una mayor parte de tu vida en línea.
```

Comment:
Mistranslation. "durante una mayor parte" ignores the "places" element. We want users to know that no matter "where" they browser, they're protected.